### PR TITLE
Better error message when missing entry file 

### PR DESCRIPTION
**Pull request by petrbela**
## Summary

If the entry file has been renamed (e.g., to `index.ts`), the script currently fails silently and prints out that the bundle file does not exist.

This change will print the correct error message instead.

## Changelog

[iOS] [Fixed] - Better error message when missing entry file 

## Test Plan

N/A

### CODE REVIEW

**Comment by petrbela** [comment]
The failing tests seem to be unrelated... Is there anything I need to do about them?

**Comment by thymikee** [comment]
You can rebase to latest master and see if that helps. But anyway, now is the moment we wait for a Facebook employee to import it and verify

File: scripts/react-native-xcode.sh [hunk 1]
```diff
@@ -67,9 +67,14 @@ if [[ "$ENTRY_FILE" ]]; then
...
-   ENTRY_FILE=${1:-index.ios.js}
- else
-   ENTRY_FILE=${1:-index.js}
+  ENTRY_FILE=${1:-index.ios.js}
+else
+  ENTRY_FILE=${1:-index.js}
+fi
+
+if [[ $DEV != true && ! -f "$ENTRY_FILE" ]]; then
+  echo "error: Entry file $ENTRY_FILE does not exist. If you use another file as your entry point, pass ENTRY_FILE=myindex.js" >&2
```
Comment (kiranjd) [inline scripts/react-native-xcode.sh:76]: I think it's a typo. 
It should have been: `"Error: entry file...`

Comment (petrbela) [reply scripts/react-native-xcode.sh:76]: Not sure what should be correct but `error: File ...` case is already used towards the end of this file so I followed the same pattern.

**Comment by kiranjd** [comment]
I think you don't need to capitalize the word after colon(:)

Kiran JD

**Pull request closed without merging**